contracts/GNUmakefile

```diff
@@ -43,7 +43,7 @@ mockery: $(mockery) ## Install mockery.
 
 .PHONY: foundry
 foundry: ## Install foundry.
-	foundryup --version nightly-de33b6af53005037b463318d2628b5cfcaf39916
+	foundryup --version nightly-515a4cc8aba1627a717a1057ff4f09c8cd3bf51f
```

Comment:
q: does the CI action now use this version automatically? Or do we still need to update something in the CI file? Previously we had an `env` for `FOUNDRY_VERSION` in `solidity-foundry.yaml`, but looks like we don't have it anymore

Reply:
Yeah, there's magic now! It actually reads this file now

Reply:
Nice!